docs/meshstack.aws.metering.md

```diff
@@ -253,3 +154,152 @@ If you would like to see the AWS costs attributed to a seller in the [chargeback
     }
 }
 ```
+
+## AWS Cost & Usage Report based approach (Deprecated)
+
+meshStack imports metering data from [AWS Cost and Usage Reports](https://aws.amazon.com/aws-cost-management/aws-cost-and-usage-reporting/)
+
+An AWS Cost and Usage Report should be created as specified in the [documentation](https://docs.aws.amazon.com/cur/latest/userguide/cur-create.html) with the following properties
+
+* Daily time granularity
+* gzip compression
+* txt/csv file format
+* Should include resource ids
+* Versioning should be setup to overwrite existing report
+
+**Note**: If an S3 bucket is not already set up, it should be created as specified in [AWS documentation](https://docs.aws.amazon.com/cur/latest/userguide/cur-s3.html)
+
+The following parameters are required to configure meshStack to process the AWS Cost and Usage Reports [[More Details](#configuration-reference)]
+
+* AWS credentials that can access the S3 bucket where the reports are written
+* The name of the S3 bucket where the report is written
+* The region of the S3 bucket
+* The name of the report
```

Comment:
is there a standard name for the report?

Reply:
No. Whoever sets up the report in the AWS console can choose the name.